README.md

```diff
@@ -152,31 +167,83 @@ These steps are relevant for javascript lambda functions, and assume you have a
     }
     ```
 
+    Your `tasks.json` file should now look similar to this:
+    
+    ```json
+    {
+        "version": "2.0.0",
+        "tasks": [
+
+            <In here you may have 0 or more additional task entries>
+
+            {
+                "label": "launchLambdaFunction",
+                ... remainder of launchLambdaFunction entry would be here, omitted for brevity ...
```

Comment:
(optional) Personally, I would prefer shorter/simpler messages throughout this PR. Things like "you may have 0 or more" feel appropriate for a spec, but noisy for a guide.